Jellyfin.Api/Controllers/ItemsController.cs

```diff
@@ -238,7 +244,17 @@ public class ItemsController : BaseJellyfinApiController
             [FromQuery] bool enableTotalRecordCount = true,
             [FromQuery] bool? enableImages = true)
         {
-            var user = userId.Equals(default) ? null : _userManager.GetUserById(userId);
+            var auth = await _authContext.GetAuthorizationInfo(Request).ConfigureAwait(false);
+
+            var user = !auth.IsApiKey && userId.HasValue && !userId.Equals(Guid.Empty)
+                ? _userManager.GetUserById(userId.Value)
+                : null;
+
+            if (!auth.IsApiKey && user is null)
```

Comment:
Should we check for empty Guid?

Reply:
If you mean `userId` I think that's handled here because that will cause `user` to be null. Or did you mean to check some other field for empty guid?

Reply:
@Bond-009 thoughts?